apis/python/src/tiledbsoma/io/registration/__init__.py

```diff
@@ -0,0 +1,113 @@
+"""
+Support for soma_joinid remapping for append-mode ingestion.
+
+This is an internal-use class; none of it is user-facing API.
```

Comment:
An internal-use module? If so, might be worth naming it `tiledbsoma.io._registration`.

Reply:
@thetorpedodog as noted in this PR's description field as well as #1278 this PR is one in a series. There is user-facing (simple, but user-facing) API content in `tiledbsoma.io.registration` coming up.

Reply:
OK, but this is a module, not a class. And given that this will be exporting public-facing APIs, and it *is* exporting several classes/functions, it is kind of weird to have "none of it is user-facing API" in the docstring (is it temporary in case we do a release with this included but not finished?). Whatever the goal is is fine, just the docstring and the code itself are sending mixed messages.

Reply:
@thetorpedodog and I want to defer the leading-underscore, exports, etc -- visibility questions -- until a follow-on PR wherein there _is_ user-facing content (there isn't any yet) and it will be clearer what needs to happen where.

I appreciate and value your feedback. I want to action this particular feedback-point on a follow-on PR.